.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -20,10 +20,7 @@ A clear and concise description of what you expected to happen.
  - Environment location: [Bare-metal, Docker, Cloud(specify cloud provider)]
  - Method of cuDF install: [conda, Docker, or from source]
    - If method of install is [Docker], provide `docker pull` & `docker run` commands used
-
-<details><summary>Click here to see environment details</summary><pre>
-Please run the `cudf/print_env.sh` script to gather relevant environment details and paste the output inside the `<details> </details>` tags.
-</pre></details>
+   - Please run and attach the output of the `cudf/print_env.sh` script to gather relevant environment details
```

Comment:
Should it be "paste"? or "attach"?  I suppose it doesn't really matter?

Reply:
I prefer paste myself, but I reverted to the previous copy to minimize the PR surface area.

Reply:
I think paste works

Also, I believe your indenting is +2 more spaces than necessary

Reply:
@mike-wendt ah so this is tough -- if I only indent 4 spaces and they paste the `<details>` tag on the same line as the second markdown bullet point, it won't be parsed as details and will look like junk. But if they paste it on a new line, it'll be doubly-nested (with 2 horizontal scrollbars), but at least still collapsed.

Reply:
I updated this gist with examples of the different indentations people might copy/paste: https://gist.github.com/trxcllnt/4843c15a483de805e7b39db740014271

Reply:
@trxcllnt this wasn't about the pasting of the `<details>` tag but how it renders in the issue, as you can see here: https://github.com/rapidsai/cudf/blob/ec4d57d39a29fd5baa14f5526dc434c3e23e9a5a/.github/ISSUE_TEMPLATE/bug_report.md

`Please run ...` is a step and should be at the top level of the list to match the other requested actions

Reply:
The way it is now, it is nested and I believe it impacts the readability

Reply:
At this point I think we should break this off into it's own section so it is clearly visible and we don't miss the other pieces of info requested above accidentally